tracing-subscriber/tests/event_enabling.rs

```diff
@@ -0,0 +1,37 @@
+#![cfg(feature = "registry")]
+
```

Comment:
it seems like it might also be good to have some tests here that:
 - `event_enabled` isn't called if `enabled` returns `false`
 - `event_enabled` returning `false` will prevent `event` from being called
 
these properties are kind of trivially obvious with the current code, but it seems worth having the tests to guard against some future change accidentally breaking it? not a hard blocker, but could be nice to have.

Reply:
agreed; added these tests